CODEOWNERS

```diff
@@ -161,8 +161,7 @@
 # - @cilium/proxy:
 #   Review low-level implementations used to redirect and process traffic at
 #   Layer 7, including via the Cilium DNS proxy and Envoy. Maintain the
-#   configurations for Envoy via xDS protocols as well as the extensible
-#   proxylib framework for Go-based layer 7 filters.
```

Comment:
nit: I would think we can still use team @cilium/proxy for review on https://github.com/cilium/proxy, no?

Reply:
oh right, I was under the impression that codeowner is managed per repo. 

Let me revert this update.